src/driver/xen.rs

```diff
@@ -161,6 +163,14 @@ impl Introspectable for Xen {
                 limit: 0,
                 selector: 0,
             },
+            idt: SystemTableReg {
```

Comment:
Same here.

Reply:
I don't think this one is possible, since you don't know which table you are building in the From impl (idtr/gdtr)
https://xenbits.xen.org/people/ianc/docs/structhvm__hw__cpu.html

Reply:
Yeah, you're right. I totally forgot that we have one large struct here with all the information regarding registers.